twyla/service/configuration.py

```diff
@@ -12,28 +12,21 @@ def from_env(prefix):
 
     for key, value in os.environ.items():
         if key.startswith(prefix):
-            from_env[key[len(prefix):].lower()] = value
+            shorter_key = key[len(prefix):]
+            from_env[shorter_key] = value
+            from_env[shorter_key.lower()] = value
```

Comment:
Why are there two key-value pairs (one with original case and the other lowercased)?

Reply:
Because some services store their configurations in upper case and some in lower. I know it's not a good idea to set two values for one key, but I couldn't come up with a better solution for the current situation.